src/Components/Web/src/Forms/EditForm.cs

```diff
@@ -36,7 +37,19 @@ public EditForm()
         /// also supply <see cref="Model"/>, since the model value will be taken
         /// from the <see cref="EditContext.Model"/> property.
         /// </summary>
-        [Parameter] public EditContext? EditContext { get; set; }
+        [Parameter]
+        public EditContext? EditContext
+        {
+            get => _fixedEditContext;
+            set
+            {
+                if (value != null)
```

Comment:
I'm not sure if this was the right thing to do now I'm looking at it here. This would mean that once an EditContext has been set it can't be set back to null. Is that a scenario that should be catered for?

Reply:
It's never going to be, is it? even if you set a model, wouldn't that be the case?

Reply:
I can't think of a reasonable scenario where you would pass in an EditContext and then later decide to set it to null. But just wanted to check

Reply:
I have the same feeling, but @SteveSandersonMS is the expert on this area, so I would defer the final say to him

Reply:
If you want a null check, could you throw `ArgumentNullException`? I think we shouldn't ignore the null value. Either accept it, or throw :)

Reply:
(Of these options, I'd prefer throwing, since there's no real value in letting people set it back to null)

Reply:
I added this check so that `_hasSetEditContextExplicity` doesn't get set to true when the value is null. Perhaps I should adjust the code to this:

```csharp
public EditContext? EditContext
{
    get => _fixedEditContext;
    set
    {
        _fixedEditContext = value;
        _hasSetEditContextExplicity = value != null ? true : false;
    }
}
```

Reply:
Looks like that would work to me.

Reply:
Ok I've push the update